config/initializers/globalize.rb

```diff
@@ -0,0 +1 @@
+Globalize.fallbacks = {es: [:es, :en]}
```

Comment:
Just so I understand it correctly, could you elaborate on the reasons why it's better to show English translations instead of empty content when the locale is set to Spanish and there's no Spanish translation?

Reply:
Sure. I'm not sure this is the best solution, but basically with this change I'm trying to maintain the same developer experience we have right now in master.

For example, this is what the homepage looks like right now in master the first time you visit the application after generating the seed data:

![selection_119](https://user-images.githubusercontent.com/7111622/44656642-f0ff1580-a9f9-11e8-9b2e-fac2525a4ec0.png)

This is what it looks like after applying #1588 (without fallbacks):

![selection_118](https://user-images.githubusercontent.com/7111622/44656658-03794f00-a9fa-11e8-986d-e4a8a898fb53.png)

By setting `:en` as a fallback locale, we get the same result as in master.

An alternative solution would be to translate all seed content to Spanish.